regression_and_census.R

```diff
@@ -9,9 +9,166 @@
 # old code-  I M P O R T I N G   C E N S U S   D A T A ---------------------------------------
 
 library(latticeExtra)
-install.packages("gee")
+library(gee)
+library(geepack)
+library(MASS)
+library(broom)
 # tidyverse
 library(dplyr)
+library(readr)
+library(tidyr)
+library(magrittr)
+library(stringr)
+library(purrr)
+# plotting libraries
+library(ggplot2)
+library(ggmap)
+library(gridExtra)
+
+
+
+# Load data ---------------------------------------------------------------
```

Comment:
All data loading code should be at the top of the file, and should not be duplicated in different sections in the same R source file.